test/javascript/packs/utils/calendar.test.js

```diff
@@ -24,4 +27,63 @@ describe("calendar", () => {
       });
     });
   });
+
+  describe("current", () => {
+    it('retunrs current time in UTC', () => {
+      expect(current().isUTC()).toBe(true)
+    })
+  })
+
+  describe("prev", () => {
+    const baseDate = moment('2022-08-01').utc()
+    it('returns previous month in same day', () => {
+      const prev_month = prev(baseDate)
+      expect(formattedDate(prev_month)).toBe('2022-07-01')
+    });
+  });
+
+  describe("next", () => {
+    const baseDate = moment('2022-08-01').utc()
+    it('returns next month in same day', () => {
+      const next_month = next(baseDate)
+      expect(formattedDate(next_month)).toBe('2022-09-01')
+    });
+  });
+
+  describe("innerRange", () => {
+    const currentDate = moment('2022-08-15').utc()
+    describe('when the base date month is the same as current date', () => {
+      const baseDate = moment('2022-08-01').utc()
+      it('returns a range from base date until current date', () => {
+        const [from, to] = innerRange(baseDate, currentDate)
+        expect(formattedDate(from)).toBe('2022-08-01')
+        expect(formattedDate(to)).toBe('2022-08-15')
+      });
+
+      describe("when called without currentDate", () => {
+        const baseDate = current().subtract(1, 'd')
+
+        it("uses current system date", () => {
+          const [_, to] = innerRange(baseDate);
```

Comment:
We don't need the `_` variable:
```suggestion
          const [, to] = innerRange(baseDate);
```